spec/models/article_spec.rb

```diff
@@ -613,5 +613,33 @@ def build_and_validate_article(*args)
     end
   end
 
+  describe "#active_published_articles" do
+    let(:not_archived_not_published_article) { create(:article, archived: false, published: false) }
+    let(:archived_not_published_article) { create(:article, archived: true, published: false) }
+    let(:not_archived_published_article) { create(:article, archived: false, published: true) }
+    let(:archived_published_article) { create(:article, archived: true, published: true) }
```

Comment:
These could probably be `let_it_be` or `let!` since they are not modified in the tests